src/pages/messages/components/MessageList.tsx

```diff
@@ -47,22 +53,26 @@ const MessageItem = ({ message }: MessageItemProps) => {
 
   return (
     <li>
-      <Typography sx={{ display: 'flex', gap: '0.25rem' }}>
-        <span>
-          {senderQuery.isLoading ? (
-            <Skeleton sx={{ width: '8rem' }} />
-          ) : (
-            <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
-              {senderName ? senderName : <i>{t('common.unknown')}</i>}
-            </b>
-          )}
-        </span>
-        <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>
-          ({new Date(message.createdDate).toLocaleString()})
-        </span>
-      </Typography>
+      <Box sx={{ bgcolor: ticketColor[ticketType], p: '0.5rem', borderRadius: '4px', maxWidth: '20rem' }}>
+        <Typography flexWrap="wrap" sx={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
naturlig at denne også ligger i sx?
```suggestion
        <Typography sx={{ display: 'flex', justifyContent: 'space-between', flexWrap:"wrap" }}>
```